cdap-ui/app/cdap/components/PipelineDetails/store/index.js

```diff
@@ -16,12 +16,13 @@
 
 import {createStore, compose} from 'redux';
 import {defaultAction} from 'services/helpers';
-import GLOBALS from 'services/global-constants';
+import {GLOBALS} from 'services/global-constants';
```

Comment:
I might have missed this before but if you happen to rebase against latest develop we can now use `composeEnhancers` from the `services/helpers` module. This is related to a change at the end of the file where we call this for enabling debugging with redux extension in chrome.

Just something for us to avoid having to repeat.